snippets/base/admin/adminmodels.py

```diff
@@ -335,7 +335,8 @@ class FxASignupTemplateInline(admin.StackedInline):
                 'utm_term',
                 'utm_campaign',
                 'block_button_text',
-                'do_not_autoblock'
+                'do_not_autoblock',
+                'retry_button_label',
```

Comment:
I didn't expect this in FxA template, since we have no `error_text` I don't think we display any error page, thus no need to the retry button. Can you verify on the ASR side if we ever display an error page?

Reply:
Unfortunately there's a bit of an explanation here.

tl;dr - I think you're right that we can omit the `retry_button_label` for FxA snippets.

The reason I included it is because on the ASR side, all snippets that follow the scene1/scene2/form submit pattern inherit from the same base template. This base template contains the retry button, so if any inheriting template fails a form submission, the base template renders the retry button.

We do not currently display an error state for FxA snippets, as the form submits directly to `https://accounts.firefox.com/`. As long as the FxA snippet template always submits to a new URL (and not back to the snippet), we can omit the `retry_button_label`.

In my ASR PR, I added a default value of "Try again" to this label, so if the inheriting template doesn't specify text, we'll have a reasonable fallback. 

If the FxA snippets ever change and submit back to themselves (to hit an FxA API or something), we may want to revisit and add `retry_button_label` to the admin. But for now, I think we're good skipping it.